duckduckgo/Android
# Hotfix/5.139.2

Task/Issue URL: https://app.asana.com/0/488551667048375/1203170357913285/f

### Description
This PR fixes a crash where AppTP crashes the app because the LRU cache is not initialized when used.

### Steps to test this PR

_Feature 1_

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203170357913285
